docs/044-ovos_installer.md

```diff
@@ -0,0 +1,13 @@
+# OVOS Installer
+
+[OVOS-Installer](https://github.com/OpenVoiceOS/ovos-installer) is the newest way to install OVOS and is probably the easiest way to install on an existing system.
+
+You do need a few packages installed on your host system before starting.
+- `curl`
+- `git`
+- `sudo` and sudo access for your user
+After installing those packages, simply run a command and follow the on screen instructions.
+
+`sh -c "curl -s https://raw.githubusercontent.com/OpenVoiceOS/ovos-installer/main/installer.sh -o installer.sh && chmod +x installer.sh && sudo ./installer.sh && rm installer.sh"`
```

Comment:
This isn't the documents' fault, but we probably shouldn't be in the "users, run this long sequence of terminal commands and execute this script with sudo" club. Let's make other ways of shipping the installer a priority